### ISSUE TITLE
List of synonyms coming along with MeiliSearch

### ISSUE THREAD

**Issue by andresribeiro** [meilisearch/MeiliSearch#883]
**Is your feature request related to a problem? Please describe.**
No, it's just a feature that could be added to improve the results

**Describe the solution you'd like**
A list of the main synonyms (like written (one, two...) and unwritten (1, 2...) numbers) that come with MeiliSearch by default, improving search results. They can be added and supported by the community, including list with different languages ...

**Describe alternatives you've considered**
If the list cannot come by default, it can be hosted on this GitHub repository, so anyone who wants to use these synonyms can clone a copy of the file containing the list

**Additional context**
If someone search by _John Wick: Chapter 2_ or _John Wick: Chapter Two_ the results would be the same without the necessity of the developer create a synonyms list

**Comment by LegendreM** [comment]
Hello @andresribeiro, your  proposal is interesting,
I think it would be more relevant to store it with a direct link in [documentation](https://docs.meilisearch.com/guides/advanced_guides/synonyms.html#the-one-way-association) as CI builded assets, what do you think @bidoubiwa ?

**Comment by bidoubiwa** [comment]
Absolutely, It would be a great addition to our [ressources](https://docs.meilisearch.com/resources/).
